samples/CommunityToolkit.Maui.Sample/MauiProgram.cs

```diff
@@ -12,7 +12,7 @@ public static MauiApp CreateMauiApp()
 	{
 		var builder = MauiApp.CreateBuilder();
 		builder.UseMauiApp<App>();
-
+		builder.UseCommunityToolkit();
```

Comment:
As far as I remember we want to avoid it.

Reply:
This is something inevitable because we need to register handlers for views that will be ported from Renderers

Reply:
Yup, it's fine to keep it. We'll be using it for the XMLNS Proposal anyways.